### ISSUE TITLE
`Base.unique!` type piracy breaks subsequent import of Plots

### ISSUE THREAD

**Issue by yha** [SciML/DataDrivenDiffEq.jl#208]
On julia 1.6.0 on Windows:
```
julia> using DataDrivenDiffEq

julia> using Plots
ERROR: InitError: ArgumentError: range must be non-empty
Stacktrace:
  [1] maximum(r::Base.OneTo{Int64})
    @ Base .\range.jl:618
  [2] unique!(b::Vector{String}, simplify_eqs::Bool)
    @ DataDrivenDiffEq ~\.julia\packages\DataDrivenDiffEq\HezmF\src\basis.jl:162
  [3] unique!(b::Vector{String})
    @ DataDrivenDiffEq ~\.julia\packages\DataDrivenDiffEq\HezmF\src\basis.jl:160
  [4] macro expansion
    @ ~\.julia\packages\JLLWrappers\bkwIo\src\wrapper_generators.jl:46 [inlined]
  [5] __init__()
    @ Ogg_jll ~\.julia\packages\Ogg_jll\2Z4tK\src\wrappers\x86_64-w64-mingw32.jl:14
  [6] _include_from_serialized(path::String, depmods::Vector{Any})
    @ Base .\loading.jl:674
  [7] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
    @ Base .\loading.jl:760
  [8] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
    @ Base .\loading.jl:689
  [9] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
    @ Base .\loading.jl:749
 [10] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
    @ Base .\loading.jl:689
 [11] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
    @ Base .\loading.jl:749
 [12] _require(pkg::Base.PkgId)
    @ Base .\loading.jl:998
 [13] require(uuidkey::Base.PkgId)
    @ Base .\loading.jl:914
 [14] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:901
during initialization of module Ogg_jll
```
seems like the issue is with
```
julia> unique!(String[])
ERROR: ArgumentError: range must be non-empty
Stacktrace:
 [1] maximum(r::Base.OneTo{Int64})
   @ Base .\range.jl:618
 [2] unique!(b::Vector{String}, simplify_eqs::Bool)
   @ DataDrivenDiffEq ~\.julia\packages\DataDrivenDiffEq\HezmF\src\basis.jl:162
 [3] unique!(b::Vector{String})
   @ DataDrivenDiffEq ~\.julia\packages\DataDrivenDiffEq\HezmF\src\basis.jl:160
 [4] top-level scope
   @ REPL[5]:1
```
Without `DataDrivenDiffEq` loaded, `unique!(String[])` works correctly.
EDIT: I've now found that it also breaks `Pkg.update()`

**Comment by nxtruong** [comment]
I second this issue. At least since Julia 1.6 (both 1.6.0 and 1.6.1) on MacOS, `DataDrivenDiffEq` (its `unique!`) broke various other packages, not just `Plots`, if these packages are loaded after.

**Comment by ChrisRackauckas** [comment]
https://github.com/SciML/DataDrivenDiffEq.jl/pull/207 will get merged soon.

**Comment by AlCap23** [comment]
Resolved via #207
